src/utils/hooks/useFormPersister.ts

```diff
@@ -0,0 +1,25 @@
+import { useDispatch } from 'react-redux';
+import { updateResourceDescriptionFormData } from '../../redux/actions/formActions';
+import useLocalStorage from './useLocalStorage';
+import { useEffect } from 'react';
+
+export default function useFormPersister(formKey: string, initialValue: object = {}) {
+  // Initiate localStorage, with a similar structure for keys as used in Redux
+  const lsKey = `forms.${formKey}`;
+  const [lsValue, setLsValues] = useLocalStorage(lsKey, initialValue);
```

Comment:
skriv gjerne ut localStorage isteden for ls slik at det blir mer eksplisitt hva det er snakk om